lib/modulr/resources/payments/payment.rb

```diff
@@ -17,6 +17,14 @@ class Payment < Base
         def initialize(raw_response, attributes = {})
           super(raw_response, attributes)
           @details = parse_details(attributes)
+          @network = parse_network(attributes[:details][:currency])
+        end
+
+        private def parse_network(currency)
+          currency = currency.upcase
+
+          return "SEPA" if currency == "EUR"
+          return "FPS" if currency == "GBP"
```

Comment:
We are using `"UK_FPS"`, so maybe this string should be named the same way.

Check `app/domains/banking/services/processors/faster_payments/uk_fps.rb:9`